.Rprofile

```diff
@@ -1,5 +1,9 @@
 source("renv/activate.R")
+box::use(logger[log_threshold])
```

Comment:
Yeah, just move all of this and other new code into the block checking if `{box}` installed.